packages/libs/client/README.md

```diff
@@ -185,7 +185,7 @@ const unsignedTransaction = Pact.builder
       decimal: '231',
     }),
   ])
-  .setMeta({ chainId: '1', senderAccount: 'your-pubkey' })
```

Comment:
Nice catch. To prevent people to use `k:` with `publicKey`, let's name it "coin-account". This then represents anything that can be an account, not just a `k:some-account`